pkg/volume/gce_pd/gce_util.go

```diff
@@ -137,7 +137,7 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (volum
 	requestBytes := c.options.Capacity.Value()
 	// GCE works with gigabytes, convert to GiB with rounding up
 	requestGB := volume.RoundUpSize(requestBytes, 1024*1024*1024)
-	err = cloud.CreateDisk(name, int64(requestGB))
+	err = cloud.CreateDisk(name, "", int64(requestGB))
```

Comment:
As per previous comment, I think you can pass in cloud.GetZone() instead of the ugly empty string here.